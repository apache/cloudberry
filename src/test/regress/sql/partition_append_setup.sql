-- start_ignore
DROP DATABASE IF EXISTS partition_append;
-- end_ignore

create database partition_append;
