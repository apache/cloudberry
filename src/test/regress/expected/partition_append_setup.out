-- start_ignore
DROP DATABASE IF EXISTS partition_append;
NOTICE:  database "partition_append" does not exist, skipping
-- end_ignore
create database partition_append;
